pkgs/stdenv/darwin/default.nix

```diff
@@ -281,6 +292,15 @@ in rec {
         ncurses libffi zlib gmp pcre gnugrep
         coreutils findutils diffutils patchutils;
 
+      # Hack to make sure we don't link ncurses in bootstrap tools. The proper
+      # solution is to avoid passing -L/nix-store/...-bootstrap-tools/lib,
+      # quite a sledgehammer just to get the C runtime.
```

Comment:
Yes eventually let's change bootstrap tools to keep builds separate rather than going FHS for no reason whatsoever!